mods.json

```diff
@@ -51321,5 +51329,1453 @@
 		"tp2": "Infamy"
+	},
+	{
+		"name": "True Aec'Letec",
+		"description": "Ces deux fichiers à placer dans l'override rendent Aec'Letec plus puissant.",
+		"urls": [
+			"https://forums.beamdog.com/discussion/40699"
+		],
+		"categories": [
+			"Gameplay"
+		],
+		"games": [
+			"BGEE",
+			"EET"
+		],
+		"authors": [
+			"Tresset"
+		],
+		"team": [],
+		"notes": [],
+		"translation_state": "n/a",
+		"safe": 1,
+		"languages": [],
+		"status": "active",
+		"last_update": "2015-04",
+		"tp2": "non-weidu"
+	},
+	{
+		"name": "Heart of Fury mode emulator",
+		"description": "Ce mod recrée les règles de Heart of Fury dans BGEE. Pourquoi utiliser ce mod alors que BGEE inclut le mode de difficulté Legacy of Bhaal ?. Premièrement parce que ce mod autorise à changer la difficulté en jeu, ce que LOB n'autorise pas. Deuxièmement parce que l'auteur à posté dans le même sujet un mod en version \"lite\" dans lequel les points de vie des monstres sont divisés par deux.",
+		"urls": [
+			"https://forums.beamdog.com/discussion/33942"
+		],
+		"categories": [
+			"Gameplay"
+		],
+		"games": [
+			"BGEE"
+		],
+		"authors": [
+			"Tisamon"
+		],
+		"team": [],
+		"notes": [],
+		"translation_state": "n/a",
+		"safe": 1,
+		"languages": [],
+		"status": "active",
+		"last_update": "2014-08",
+		"tp2": "HoFemu"
+	},
+	{
+		"name": "Short Sword of Backstabbing (PnP)",
+		"description": "Ce mod corrige l'Epée courte d'attaque sournoise pour quelle corresponde à ses caractéristique pnp. Elle devient une arme +2 qui applique des bonus supplémentaires si elle est portée par un voleur : +2 supplémentaire au toucher et +1 au modificateur d'attaque sournoise.",
+		"urls": [
+			"https://forums.beamdog.com/discussion/64157"
+		],
+		"categories": [
+			"Sort et objet"
+		],
+		"games": [
+			"BGEE"
+		],
+		"authors": [
+			"Greener"
+		],
+		"team": [],
+		"notes": [],
+		"translation_state": "no",
+		"safe": 2,
+		"languages": [
+			"en"
+		],
+		"status": "active",
+		"last_update": "2017-08",
+		"tp2": "SSoB2"
+	},
+	{
+		"name": "Ōdachi",
+		"description": "Ce mod ajoute dans le jeu un katana à deux mains.",
+		"urls": [
+			"https://forums.beamdog.com/discussion/65872"
+		],
+		"categories": [
+			"Sort et objet"
+		],
+		"games": [
+			"BGEE"
+		],
+		"authors": [
+			"Greener"
+		],
+		"team": [],
+		"notes": [],
+		"translation_state": "no",
+		"safe": 2,
+		"languages": [
+			"en"
+		],
+		"status": "missing",
+		"last_update": "",
+		"tp2": ""
+	},
+	{
+		"name": "Ellyrian",
+		"description": "Ce mod ajoute Ellyrian une clerc/rôdeuse recrutable au Jongleur Jovial dans Beregost. Elle a une mission et n'est pas garantie sans bugs.",
+		"urls": [
+			"https://forums.beamdog.com/discussion/32979"
+		],
+		"categories": [
+			"PNJ One Day"
+		],
+		"games": [
+			"BGEE"
+		],
+		"authors": [
+			"CaloNord"
+		],
+		"team": [
+			"ggallegher",
+			"Isandir"
+		],
+		"notes": [
+			"Regardez en page 2 du sujet, le même mod avec un set de voix est proposé."
+		],
```

Comment:
Excellent la note, dans ce cas on pourrait même ajouter l'url complet vers le commentaire : `https://forums.beamdog.com/discussion/32979/ellyrian-the-cleric-ranger-npc/p2#Comment_618788`